src/marqo/tensor_search/tensor_search.py

```diff
@@ -1756,12 +1872,12 @@ def get_query_vectors_from_jobs(
 
 def get_content_vector(possible_jobs: List[VectorisedJobPointer], job_to_vectors: Dict[JHash, Dict[str, List[float]]],
                        jobs: Dict[JHash, VectorisedJobs],
-                       treat_urls_as_images: bool, content: str) -> List[float]:
+                       treat_urls_as_media: bool, content: str) -> List[float]:
```

Comment:
`treat_urls_as_media` is unused. Why? You can use your IDE to find instnaces of unused variables as there are quite a few, and investigate these

Reply:
This was just leftover code. Will delete this one and investigate other ones.

Reply:
done